emv2/org.osate.xtext.aadl2.errormodel/src/org/osate/xtext/aadl2/errormodel/validation/ErrorModelValidator.java

```diff
@@ -974,10 +974,20 @@ private void checkTransitionTargetTypes(ErrorBehaviorTransition ebt) {
 			if (tt == null || tt.getTypeTokens().isEmpty()) {
 				return;
 			}
+			if (tt.getTypeTokens().size() > 1) {
+				error(ebt,
+						"Target state " + ebs.getName()
+								+ " should be associated with only a single error type instead of "
+								+ tt.getTypeTokens().size() + " error types");
+			}
```

Comment:
Change to  " may only have a singe error type"
_should_ is too weak.
The _instead of ..._ part is unnecessary.